doc/run/repl.md

```diff
@@ -79,9 +80,9 @@ Congratulations! Now you have successfully completed a session using SQLFlow syn
 
 |             Option                      | Description |
 |-----------------------------------------|-------------|
+| -sqlflow_server \<quoted-query-string\>              | Specify sqlflow server address, in `host:port` form, e.g. `-sqlflow_server "localhost:50051"` |
 | -e \<quoted-query-string\>              | Execute from command line without entering interactive mode. e.g. <br>`-e "SELECT * FROM iris.train TRAIN DNNClassifier..." `<br>does the same thing as the training example above.|
-| -f \<filename\>                         | Execute from file without entering interactive mode. e.g. <br>`-f ./my_sqlflow.sql`<br>does the same thing as<br>`< ./my_sqlflow.sql` and `cat ./my_sqlflow.sql \| REPL...` |
-| -model_dir \<local-directory\>          | Save model to a local directory. e.g. `-model_dir "./models/"` |
+| -f \<filename\>                         | Execute from file without entering interactive mode. e.g. <br>`-f ./my_sqlflow.sql`<br>does the same thing as<br>`< ./my_sqlflow.sql` and `cat ./my_sqlflow.sql \| sqlflow...` |
 | -datasource \<database-connection-url\> | Connect to the specified database. e.g. `-datasource "mysql://root:root@tcp(host.docker.internal:3306)/" ` |
```

Comment:
`-datasource` and `-sqlflow_server` can be configured in `~/.sqlflow_env`, please add an instruction on this.

Reply:
done